frontend/src/component/archive/ArchiveTable/ArchivedFeatureActionCell/ArchivedFeatureDeleteConfirm/ArchivedFeatureDeleteConfirm.tsx

```diff
@@ -36,6 +36,9 @@ export const ArchivedFeatureDeleteConfirm = ({
     const { setToastData, setToastApiError } = useToast();
     const { deleteFeatures } = useProjectApi();
 
+    const singularOrPluralToggles =
```

Comment:
One day when we do i18n you're gonna hate yourself :P

Reply:
Depending on how we do pluralization this would be much easier and baked in.

Example:
https://kazupon.github.io/vue-i18n/guide/pluralization.html#accessing-the-number-via-the-pre-defined-argument